cards.css

```diff
@@ -0,0 +1,147 @@
+* {
+  box-sizing: border-box;
+  /* font-family: 'PT Sans', sans-serif; */
+  font-family: "Cairo", sans-serif;
+}
+.text-center {
+  text-align: center;
+}
+.ca-content {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+.card {
+  width: 20rem;
+  border: 1px solid rgb(205 198 198);
+  border-radius: 5px;
+  overflow: hidden;
+  background: rgb(243, 242, 242);
+}
+.ca-lg {
+  width: 100%;
+}
+.ca-nobg {
+  background-color: #fff;
+}
+.ca-bg {
+  background: rgb(243, 242, 242) !important;
+}
+.ca-shadow {
+  box-shadow: 9px 5px 5px rgb(0 0 0 / 30%);
+}
+.text-center {
+  text-align: center;
+}
+
+.ca-header {
+  padding: 8px;
+}
+
+.ca-body {
+  padding: 20px;
+  background: #fff;
+}
+
+.ca-footer {
+  padding: 8px;
+}
+
+.ca-img-overlay {
+  z-index: 2;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.card__image {
```

Comment:
Stick to the one naming convention in a project. This class follows BEM but other ones aren't.

Reply:
Ok Will Update